examples/model_compress/knowledge_distill/KnowledgeDistill.md

```diff
@@ -0,0 +1,31 @@
+Knowledge Distillation on NNI Compressor
+===
+
+## KnowledgeDistill
+
+Knowledge distillation support while fine-tuning the compressed model, in [Distilling the Knowledge in a Neural Network](https://arxiv.org/abs/1503.02531),  the compressed model is trained to mimic a pre-trained, larger model.  This training setting is also referred to as "teacher-student",  where the large model is the teacher and the small model is the student.
+
+![](../../../docs/img/distill.png)
+
+### Usage
+
+PyTorch code
+
+```python
+from knowledge_distill.knowledge_distill import KnowledgeDistill
+kd = KnowledgeDistill(kd_teacher_model, kd_T=5, kd_beta=1)
+
```

Comment:
It is hard to understand this kd_beta without reading source code of KnowledgeDistill class. and considering the `student_loss` is passed to KnowledgeDistill only for add, my suggestion:
Remove the kd_beta and student_loss parameter of KnowledgeDistill.loss,   KnowledgeDistill is responsible only the KD loss, so that we can use it like this:
```
student_loss = F.cross_entropy(output, target)
kd_loss = kd.loss(data, output)
loss = student_loss * 0.2 + kd_loss * 0.8
...
```

Reply:
resolved